addons/stock/models/stock_orderpoint.py

```diff
@@ -332,8 +332,10 @@ def _get_orderpoint_action(self):
         to_refill = defaultdict(float)
         all_product_ids = []
         all_warehouse_ids = []
+        ConfigParameter = self.env['ir.config_parameter'].sudo()
+        forecast_interval_months = ConfigParameter.get_param("stock.forecast_interval_months", "3")
         # Take 3 months since it's the max for the forecast report
-        to_date = add(fields.date.today(), months=3)
+        to_date = add(fields.date.today(), months=int(forecast_interval_months))
```

Comment:
Please update the comment above to mention something like: "By default, take 3 months ...". I think it would be good to also move that comment above the line that you've added (the one with `forecast_interval_months`).

Reply:
done! ;)